Update Mbrola links to and add configuration details

valdisvi opened the pull request.

rhdunn commented:
That github repository is not an official MBROLA distribution channel. The documentation should keep referring to the original source, even if there are no updates to it.

valdisvi commented:
Yes, but github repository has newer binaries for Linux, and I can't neither convince Thierry Dutoit to update their original website, or make Mbrola open source...

rhdunn commented:
I understand the problem you are facing. The problem I have is that there is no way to verify the authenticity of the binaries outside them being on the MBROLA website, or committed/distributed by Thierry Dutoit. The other problem is how are you going to get the linux distributions to use the github MBROLA binary for the same reason?

Distributions like Debian have provided workarounds for the issues of running 64-bit MBROLA due to libc6 changes (e.g. https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=856331, https://tracker.debian.org/pkg/mbrola).

Are there any other issues running MBROLA on Linux?

valdisvi commented:
Reece, I rewrote this commit and put links to official MBROLA site back. I added only note about latest ["develompent versions" MBROLA in github](https://github.com/espeak-ng/espeak-ng/pull/442/commits/084fb83af9e5d7acb8f0132baee8c983d1d682a1#diff-d937e90415fb7418533ab3ca3a7463ceR103).
I contacted Samuel Thibault and he updated Debian packages with rebuilt binaries (they don't support `-d` switch though). I couldn't comment Debian bug, because it is closed.
